manifests/mod/ssl.pp

```diff
@@ -3,6 +3,8 @@
   $ssl_cryptodevice                                         = 'builtin',
   $ssl_options                                              = [ 'StdEnvVars' ],
   $ssl_openssl_conf_cmd                                     = undef,
+  $ssl_cert                                                 = undef,
```

Comment:
Should this be ```Optional[String] $ssl_cert``` or even ```Optional[Stdlib::Absolutepath] $ssl_cert```? Same for ssl_key.

Reply:
updated.